source/jobs/specifying_resources.rst

```diff
@@ -165,6 +165,16 @@ In total, each of the 16 processes can use 10 GB RAM.
    users are recommended to specify the total amount of RAM requested for the job
    with ``-l mem=<total_memory>``.
 
+.. note::
+
+   On the KU Leuven/UHasselt Tier-2 cluster, setting ``pmem`` to a value
+   higher than the total amount of memory on the node divided by the number of
+   cores on the node, will lead the scheduler to reserve the entire node. This
```

Comment:
How about condensing "the total amount of memory on the node divided by the number of cores on the node" into "the available memory per core"?